docs/docs/support/general.md

```diff
@@ -0,0 +1,52 @@
+---
+title: General
+---
+
+## Support States
+
+It's important to note that support may differ depending on the feature, and not all features may be fully supported. 
+We always strive to provide the best support possible for our customers and community,
+but we may not be able to provide immediate or comprehensive support for all features.
+Also the support may differ depending on your contracts. Read more about it on our [Legal Page](/docs/legal/introduction)
+
+### Fully Supported
+
+Fully supported features are those that are guaranteed to work as intended and are fully tested by our team.
+If you encounter any issues with a fully supported feature, you can contact us, 
+and we will provide prompt resolution to any issues you may experience.
+
+### Partially Supported
+
+Partially supported features are those that we support to a certain extent, 
+but may not have been fully tested or may have some limitations.
+If you encounter issues with a partially supported feature, we will make our best effort to assist you, 
+but we may not be able to provide immediate resolution or comprehensive support.
+
+**Partially supported features**
+
+- Database PostgreSQL
+
+### Community Supported
+
+Community supported features are those that have been developed by our community and may not have undergone extensive testing or support from our team. 
+If you encounter issues with a community supported feature, we encourage you to seek help from our community or other online resources, where other users can provide assistance.
+Join our [Discrod Chat](https://zitadel.com/chat)
+
+## Software Release Cycle
+
+It's important to note that both Alpha and Beta software can have breaking changes, meaning they are not backward-compatible with previous versions of the software.
+Therefore, it's recommended to use caution when using Alpha and Beta software, and to always back up important data before installing or testing new software versions.
+
```

Comment:
```suggestion

Only features in General Availability will be covered by support services.

We encourage our community to test Alpha and Beta software and provide feedback via our [Discord Chat](https://zitadel.com/chat). 

```

Reply:
Do you think it is not relevant to state that alpha and beta can have breaking changes?
And also that its recommended to use it with caution?

Reply:
Yes. I wanted to make an addition. Does it delete something on Line 40?